plugins/serializers/registry.go

```diff
@@ -114,6 +114,9 @@ type Config struct {
 	// Output string fields as metric labels; when false string fields are
 	// discarded.
 	PrometheusStringAsLabel bool `toml:"prometheus_string_as_label"`
+
+	// Convert "_" in paths to "." for Wavefront
+	WavefrontConvertPaths bool `toml:"convert_paths"`
```

Comment:
With this change you will break _all_ existing wavefront users as suddenly (when using their old config) their paths change without further notice on updateing telegraf. This cannot happen. Please invert that setting such that by default (i.e. not set by the user) the paths are converted as before.

A natural alternative would be `wavefront_keep_path` or `wavefront_disable_path_conversion`.